src/python/director/startup.py

```diff
@@ -749,9 +757,13 @@ def toggle(self):
 
 
 imageOverlayManager = ImageOverlayManager()
-imageViewHandler = ToggleImageViewHandler(imageOverlayManager)
+imageWidget = cameraview.ImageWidget(cameraview.imageManager, 'CAMERA_LEFT', view, visible=False)
+imageViewHandler = ToggleImageViewHandler(imageWidget)
 showImageOverlay = imageOverlayManager.show
 hideImageOverlay = imageOverlayManager.hide
+showImageWidget = imageWidget.show
+hideImageWidget = imageWidget.hide
+setImageWidgetSource = imageWidget.setImage
```

Comment:
does anyone actually use these functions?  The show/hide functions were originally for convenience before we had a toggle button.  Now that we have a toggle button in the toolbar, we should just delete all these functions